modules/blake3/1.5.4.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,13 @@
+module(
+    name = "blake3",
+    version = "1.5.4.bcr.1",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_cc", version = "0.1.1")
+bazel_dep(name = "platforms", version = "0.0.11")
+
+# Required only by `--extra_toolchains=@local_config_cc//:cc-toolchain-arm64_windows` used in CI.
```

Comment:
You can remove this, those won't be visible to the root module when running the test. I applied a fix in the presubmit.yml file for windows arm64

Reply:
Done.